index.js

```diff
@@ -359,6 +363,83 @@
       [a, b, a],
       function(x, y) { return x; });
 
+  //# A :: (a -> b) -> a -> b
+  //.
+  //. The A combinator. Takes a function and a value and returns the result of
+  //. applying the function to the value. Equivalent to Haskell's `($)` function.
+  //.
+  //. ```javascript
+  //. > S.A(R.add(1), 1)
+  //. 2
+  //.
+  //. > R.map(S.flip(S.A, 3), [R.add(2), R.multiply(2)])
+  //. [5, 6]
+  //. ```
+  S.A =
+  def('A',
+      {},
+      [$.Function, a, b],
+      function(f, x) { return f(x); });
+
+  //# C :: (a -> b -> c) -> b -> a -> c
+  //.
+  //. The C combinator. Takes a curried binary function and two values and
+  //. returns the result of applying the function to the values in reverse.
+  //. Equivalent to Haskell's `flip` function.
+  //.
+  //. This function is very similar to `S.flip`, except that its first argument
+  //. must be curried. This also allows it to work with manually curried
+  //. functions.
+  //.
+  //. ```javascript
+  //. > R.gt(1, 2)
+  //. false
+  //.
+  //. > S.C(R.gt, 1, 2)
+  //. true
+  //.
+  //. > R.map(S.C(S.A, 3), [R.add(2), R.multiply(2)])
+  //. [5, 6]
+  //. ```
+  S.C =
+  def('C',
+      {},
+      [$.Function, b, a, c],
+      function(f, x, y) { return f(y)(x); });
+
+  //# B :: (b -> c) -> (a -> b) -> a -> c
+  //.
+  //. The B combinator. Takes two functions and a value and returns the result
+  //. of applying the first function to the result of applying the second to the
+  //. value. Equivalent to Haskells `(.)` function and to `S.compose`.
+  //.
+  //. ```javascript
+  //. > S.B(R.multiply(2), R.add(2), 0)
+  //. 4
```

Comment:
I'd like to use the [`S.compose`][1] example here as well:

```javascript
> S.B(Math.sqrt, S.inc, 99)
10
```

Unary functions such as `Math.sqrt` and `S.inc` are slightly less noisy than partially applied binary functions, yet convey the same idea.


[1]: https://github.com/plaid/sanctuary#compose